lib/patch_ruby/models/create_bitcoin_estimate_request.rb

```diff
@@ -41,7 +41,7 @@ def self.acceptable_attributes
     # Attribute type mapping.
     def self.openapi_types
       {
-        :'timestamp' => :'String',
+        :'timestamp' => :'Time',
```

Comment:
This is a bonus of defining the type correctly in swagger. 

Technically this could be breaking, but I'm going to make the bet that no one uses this one.